### PR TITLE
Updated container resource HPA KEP

### PR DESCRIPTION
**Pull request by arjunrn**
Minor updates to the KEP which reflect current implementation.

### CODE REVIEW

File: keps/sig-autoscaling/0001-container-resource-autoscaling.md [hunk 1]
```diff
@@ -312,8 +308,9 @@ criteria required because it will graduate when the original API graduates to `s
 For cluster upgrades the HPAs from the previous version will continue working as before. There
 is no change in behavior or flags which have to be enabled or disabled.
 
-For clusters which have HPAs which use `ContainerResourceMetricSource` or `PodResourceMetricSource`
-a downgrade is possible after HPAs which use this new source have been modified to use
-`ResourceMetricSource` instead.
+A new feature gate `HPAContainerMetrics` is added in version 1.20 which is turned of by default.
```
Comment (josephburnett) [inline keps/sig-autoscaling/0001-container-resource-autoscaling.md:311]: Spelling: "of" -> "off"

**Review by josephburnett — APPROVED** [review]
Looks good.  Thanks for including the container name migration instructions!

/lgtm

File: keps/sig-autoscaling/0001-container-resource-autoscaling.md (hunk 1 above)
Comment (arjunrn) [reply keps/sig-autoscaling/0001-container-resource-autoscaling.md]: Fixed

**Pull request merged**